cwms_radar_api/src/main/java/cwms/radar/api/RatingTemplateController.java

```diff
@@ -0,0 +1,181 @@
+package cwms.radar.api;
+
+import java.util.Optional;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javax.servlet.http.HttpServletResponse;
+
+import com.codahale.metrics.Histogram;
+import com.codahale.metrics.MetricRegistry;
+import com.codahale.metrics.Timer;
+import cwms.radar.api.errors.RadarError;
+import cwms.radar.data.dao.RatingTemplateDao;
+import cwms.radar.data.dto.rating.RatingTemplate;
+import cwms.radar.data.dto.rating.RatingTemplates;
+import cwms.radar.formatters.ContentType;
+import cwms.radar.formatters.Formats;
+import io.javalin.apibuilder.CrudHandler;
+import io.javalin.core.util.Header;
+import io.javalin.http.Context;
+import io.javalin.plugin.openapi.annotations.OpenApi;
+import io.javalin.plugin.openapi.annotations.OpenApiContent;
+import io.javalin.plugin.openapi.annotations.OpenApiParam;
+import io.javalin.plugin.openapi.annotations.OpenApiResponse;
+import org.jetbrains.annotations.NotNull;
+import org.jooq.DSLContext;
+
+import static com.codahale.metrics.MetricRegistry.name;
+import static cwms.radar.data.dao.JooqDao.getDslContext;
+
+
+public class RatingTemplateController implements CrudHandler {
+    private static final Logger logger = Logger.getLogger(RatingTemplateController.class.getName());
+    private final MetricRegistry metrics;
+
+    private static final int DEFAULT_PAGE_SIZE = 100;
+
+    private final Histogram requestResultSize;
+
+    public RatingTemplateController(MetricRegistry metrics){
+        this.metrics=metrics;
+        String className = this.getClass().getName();
+        requestResultSize = this.metrics.histogram((name(className,"results","size")));
+    }
+
+
+    private Timer.Context markAndTime(String subject)
+    {
+        return Controllers.markAndTime(metrics, getClass().getName(), subject);
+    }
+
+
+
+    @OpenApi(
+        queryParams = {
+            @OpenApiParam(name="office", description="Specifies the owning office of the Rating Templates whose data is to be included in the response. If this field is not specified, matching rating information from all offices shall be returned."),
+            @OpenApiParam(name="template-id-mask", description="RegExp that specifies the rating template IDs to be included in the response. If this field is not specified, all rating templates shall be returned."),
+                @OpenApiParam(name="page",
+                        description = "This end point can return a lot of data, this identifies where in the request you are. This is an opaque value, and can be obtained from the 'next-page' value in the response."
+                ),
+                @OpenApiParam(name="page-size",
+                        type=Integer.class,
+                        description = "How many entries per page returned. Default " + DEFAULT_PAGE_SIZE + "."
+                ),
+        },
+            responses = {
```

Comment:
Indentation here as well, if you're going to fix them.
I also just noticed here, and in the RatingSPecController.java file, that the OpenApiParam annotations above are also not aligned. I thought they were nested at first, but they don't appear to be (line 57 in this file).

Reply:
Yeah, weird.  Will fix.